.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.3-jessie-node-browsers
+    steps:
+      - checkout
+      - run: bundle install
+      - run:
+          name: "Build AsciiDoc files"
+          command: bundle exec asciidoctor src/index.adoc -D dist
+  deploy:
+    docker:
+      - image: circleci/ruby:2.3-jessie-node-browsers
+    steps:
+      - checkout
+      - run: bundle install
```

Comment:
Good _non-premature_ optimization.

We build everything in the previous job (named `build`) thus we should persist the [build in the workspace](https://circleci.com/docs/2.0/configuration-reference/#persist_to_workspace) and [attach the workspace](https://circleci.com/docs/2.0/configuration-reference/#attach_workspace) in the `deploy` job (also `checkout` step is going to be redundant).

> **Note**: Can not really give clear example since, I do not know how `bundle exec asciidoctor` really works, I assume it generates files in the `./` so example from CircleCI should be enough.

Gains:
- good practice
- no need for ruby image, and can swap to `antman-cli` image instead (if we decide to build it)

Reply:
feel free to do this if you know how. I'm not familiar with circleci this much :)